data-access-service/src/main/java/uk/gov/justice/laa/dstew/access/mapper/ApplicationSummaryMapper.java

```diff
@@ -30,6 +30,10 @@ default ApplicationSummary toApplicationSummary(ApplicationSummaryEntity applica
       app.setApplicationId(applicationSummaryEntity.getId());
       app.setApplicationStatus(applicationSummaryEntity.getStatus());
       app.setApplicationReference(applicationSummaryEntity.getApplicationReference());
+      app.setAssignedTo(applicationSummaryEntity.getCaseworker() != null 
+                        ? 
+                        applicationSummaryEntity.getCaseworker().getId() : 
+                        null);
```

Comment:
@b4barnett  at this point, is the related caseworker entity pre-loaded here or does it work like in python that fires off a query (to optimise load times) when `getSomething` is called? just wondering if we should assign to an external variable to hit the DB only once, if that's the case